### DISCUSSION TITLE
Tidak ada notifikasi

### DISCUSSION THREAD

**Discussion by candra1098** [kirimwa-id/forum#52] [Tanya Jawab]
Pada saat send message sukses dan terkirim ke perangkat tapi tidak ada notifikasinya

**Comment by rioastamal** [comment]
Tidak ada notifikasi seperti apa yang dimaksud? Pesan tidak terkirim atau seperti apa? Silahkan lihat status detilnya lewat `GET /v1/messages/MESSAGE_ID`

**Reply by candra1098** [reply]
Pesan sudah berhasil masuk mas ke wa yg di tuju. Yang saya maksud notifikasi suara yg tidak ada

**Reply by rioastamal** [reply]
@candra1098, menurut saya kalau itu tergantung konfigurasi HP tersebutdan tidak ada hubungan dengan API KirimWA.id.
